torchgeo/transforms/indices.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""TorchGeo indices transforms."""
+
+from typing import Dict
+
+import torch
+from torch import Tensor
+from torch.nn import Module  # type: ignore[attr-defined]
+
+# https://github.com/pytorch/pytorch/issues/60979
+# https://github.com/pytorch/pytorch/pull/61045
+Module.__module__ = "torch.nn"
+
+
+def ndvi(red: Tensor, nir: Tensor) -> Tensor:
```

Comment:
One of the things torchvision does that I've never understood is to have separate "functional" transforms in addition to regular transforms. Do we need these to be separate methods, or can we just have a single method?

Reply:
I believe the only reasoning behind having a functional transform is if you do not know some parameters (e.g. dynamic input shapes) until performing a forward pass. In this case though we can remove the function.

Reply:
I changed the class version of the indices transforms to `AppendNDVI`, for example. The reasoning being that `ndvi` performs the actual computation, while `AppendNDVI` is a transform which facilitates taking an input dict, computing the desired index, and appending it to the channel dimension. Let me know if this works if what naming scheme you would suggest.